.vib/owncloud/goss/goss.yaml

```diff
@@ -0,0 +1,27 @@
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: []
+file:
+  /opt/bitnami/apache/conf/extra/httpd-manual.conf:
+    mode: "0664"
+    filetype: file
+    exists: true
+  /opt/bitnami/mysql:
+    mode: "0755"
+    filetype: directory
+    exists: true
+  /bitnami/owncloud/data:
+    mode: "0770"
+    filetype: directory
+    exists: true
+  /opt/bitnami/owncloud/config/config.php:
+    mode: "0777"
+    filetype: symlink
+    exists: true
+    contains:
+      - dbtype
+      - dbuser
```

Comment:
Does this file contain the values of these parameters? If so, you could do something similar [to this test](https://github.com/FraPazGal/charts/blob/1031ef6e2dfee0d1db7a9c3ddb6339567e44bdc5/.vib/joomla/goss/goss.yaml#L18).

Reply:
Yes, but the syntax is a bit strange, look:

I'm not sure if this is possible and how. WDYT?

![Screenshot 2022-06-14 at 15 09 41](https://user-images.githubusercontent.com/47592680/173585111-2415c269-bd18-4bd4-8bde-722619d5d67b.png)

Reply:
Yes, it is possible if you use a RegEx instead of regular text.

Check https://github.com/bitnami/charts/blob/df8bd7e304600cd5851516aeb7b9adad381b79f0/.vib/prestashop/goss/goss.yaml#L31

Reply:
Mmm I think it should be possible without using a RegEx:

```suggestion
      - {{ printf "'dbuser' => '%s'" .Env.OWNCLOUD_DATABASE_USER }}
```

Nevertheless, you can also use a regex. If I'm not mistaken, it should be something like:

```suggestion
      - {{ printf "/dbuser.*%s/" .Env.OWNCLOUD_DATABASE_USER }}
```